appbuilder/services/npm-service.ts

```diff
@@ -203,8 +206,32 @@ export class NpmService implements INpmService {
 		}).future<boolean>()();
 	}
 
-	private buildNpmRegistryUrl(packageName: string, version: string): string {
-		return `${NpmService.NPM_REGISTRY_URL}/${packageName.replace("/", "%2F")}?version=${encodeURIComponent(version)}`;
+	private buildNpmRegistryUrl(packageName: string, version: string): IFuture<string> {
+		return (() => {
+			let registryUrl = this.getNpmRegistryUrl().wait();
+			return `${registryUrl}/${packageName.replace("/", "%2F")}?version=${encodeURIComponent(version)}`;
```

Comment:
There's one [more place](https://github.com/telerik/mobile-cli-lib/blob/master/services/plugins/npm-registry-plugins-source.ts#L31) we use this dreaded `.replace("/", "%2F")`. I'd prefer we extract it to a common place in case some other symbol needs to be replaced this way.

This could be done in a separate PR though